src/main/java/uk/gov/hmcts/reform/et/syaapi/controllers/PostcodeController.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.et.syaapi.controllers;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.dwp.regex.InvalidPostcodeException;
+import uk.gov.hmcts.ecm.common.model.helper.TribunalOffice;
+import uk.gov.hmcts.reform.et.syaapi.service.PostcodeToOfficeService;
+
+import java.util.Optional;
+
+import static org.springframework.http.ResponseEntity.ok;
+
+/**
+ * Used to test the features of postcode lookup.
+ */
+@RestController
+public class PostcodeController {
```

Comment:
I don't think there is any requirement to expose this functionality as a Rest endpoint. I think it should only be a dependency for Java code that needs to submit data to CCD.

Reply:
I will remove the rest controller.